tests/foreman/api/test_activationkey.py

```diff
@@ -126,6 +130,20 @@ def test_negative_create_3(self):
                         unlimited_content_hosts=True,
                     ).create()
 
+    def test_negative_create_4(self):
```

Comment:
@omaciel @oshtaier started this some time ago, will try to find the issue or will create a new one if necessary.